sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewViewModel.groovy

```diff
@@ -14,23 +15,25 @@ import life.qbic.datamodel.samples.Sample
 */
 class ProjectOverviewViewModel {
 
-    List<Project> projects = []
+    ObservableList projects = new ArrayList<Project>()
+    final ProjectResourceService projectResourceService
 
-    ProjectOverviewViewModel(){
-        generateMockData()
-    }
-
-    void generateMockData(){
-        Location location = new Location()
+    Subscriber<life.qbic.datamodel.dtos.projectmanagement.Project> projectAddedSubscription
+    Subscriber<life.qbic.datamodel.dtos.projectmanagement.Project> projectRemovedSubscription
```

Comment:
Why are those class attributes?

Reply:
Remnants from the time there were 2 project classes in the viewmodel. Nice Catch 👁️